optimade/server/config.py

```diff
@@ -260,6 +260,20 @@ class ServerConfig(BaseSettings):
             "MongoDB-based backend."
         ),
     )
+    schema_url: Optional[Union[str, AnyHttpUrl]] = Field(
+        f"https://schemas.optimade.org/openapi/{__api_version__}/optimade.json",
```

Comment:
Are you sure this line of code is correct?
If I enter  [https://schemas.optimade.org/openapi/1.1.0/optimade.json](https://schemas.optimade.org/openapi/1.1.0/optimade.json) in my browser, I get a 404 file not found message.
[https://schemas.optimade.org/openapi/v1/optimade.json](https://schemas.optimade.org/openapi/v1/optimade.json) does seem to work, and refers to OPTIMADE version 1.1.0.